tests/network-tests/src/Api.ts

```diff
@@ -807,48 +808,72 @@ export class Api {
 
   async createApp(
     memberId: u64,
-    channelId: number,
     name: string,
-    appMetadata?: IAppMetadata
+    appMetadata?: IAppMetadata,
+    ownedByLead?: boolean
   ): Promise<ISubmittableResult> {
     const memberAccount = await this.getMemberControllerAccount(memberId.toNumber())
     if (!memberAccount) {
       throw new Error('invalid member id')
     }
+    if (ownedByLead) {
+      const meta = new LeadRemarked({
+        createApp: new CreateApp({
+          name,
+          appMetadata,
+        }),
+      })
+      return this.sender.signAndSend(
+        this.api.tx.contentWorkingGroup.leadRemark(Utils.metadataToBytes(LeadRemarked, meta)),
+        memberAccount.toString()
+      )
+    } else {
+      const meta = new MemberRemarked({
+        createApp: new CreateApp({
+          name,
+          appMetadata,
+        }),
+      })
+      return this.sender.signAndSend(
+        this.api.tx.members.memberRemark(memberId, Utils.metadataToBytes(MemberRemarked, meta)),
+        memberAccount.toString()
+      )
+    }
+  }
 
-    const msg: IChannelOwnerRemarked = {
-      createApp: {
-        name,
-        appMetadata,
-      },
+  async updateApp(memberId: u64, appId: string, appMetadata: IAppMetadata): Promise<ISubmittableResult> {
+    const memberAccount = await this.getMemberControllerAccount(memberId.toNumber())
+    if (!memberAccount) {
+      throw new Error('invalid member id')
     }
 
+    const meta = new LeadRemarked({
```

Comment:
Shouldn't that depend on the app owner? (ie. `LeadRemark` for DAO-owned apps, `MemberRemark` for member-owned apps)